agential/cog/prompts/reflexion.py

```diff
@@ -4,20 +4,30 @@
 
 
 # Fewshot CoT examples for prompting ReflexionCoT agent (think/act).
-REFLEXION_COT_FEWSHOT_EXAMPLES = """Relevant Context: The Nile River is the longest river in the world, spanning approximately 6,650 kilometers (4,132 miles) in length. It flows through eleven countries in northeastern Africa, including Egypt, Sudan, and Uganda.
-Question: What is the longest river in the world?
-Thought: The question asks for the longest river in the world, which I know is the Nile River based on the context provided.
-Action: Finish[Nile River]
+REFLEXION_COT_FEWSHOT_EXAMPLES = """Relevant Context: The eastern sector of the Colorado orogeny extends into the High Plains, which is characterized by its relatively flat terrain with elevations ranging from around 1,800 to 7,000 feet.
```

Comment:
oh shoot, i had bad naming convention here. could you rename this for the specific benchmark